### PR TITLE
feat: New Aircraft Updates tab

### PR DESCRIPTION
**Pull request by Spherrrical**
New aircraft updates tab in the website (Work In Progress for Automation)

### CODE REVIEW

File: aircraft.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <a class="activeClass" href="aircraft.html">AIRCRAFT UPDATES</a>
+			
+			<a class="navbarlink" href="https://discord.gg/MdtbBnVK9Y" target="_ blank">DISCORD </a>
+			<a class="navbarlink" href="https://github.com/Pinpoint-Simulations" target="_ blank">GITHUB</a>
+			<a class="navbarlink" href="https://www.youtube.com/channel/UCLwmhnbgYkvLHCKDCsFkTJw" target="_ blank">YOUTUBE</a>
+      
+			
+    </div>
+    
+    <h1 class="title">AIRCRAFT UPDATES</h1>
```
Comment (scoopscode) [inline aircraft.html:30]: Change "AIRCRAFT UPDATES" to "UPDATES" for all progress from our projects.

File: aircraft.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    <div class="space"></div>
+    <div class="navbarfooters">
+      <a class="navbarfooter" href="tos.html">TERMS OF SERVICE</a>
+      <a class="navbarfooter" href="privacy.html">PRIVACY POLICY</a>
+      <a class="navbarfooter" href="https://github.com/Pinpoint-Simulations/website" target="_blank">SOURCE CODE</a>
+    </div>
+    
+    
+    <footer class="copy"> ©️ Pinpoint Simulations 2021</footer>
```
Comment (scoopscode) [inline aircraft.html:45]: Remove copyright (unless we are copyrighted in which I didnt know).

File: index.html [hunk 3]
```diff
@@ -17,6 +17,7 @@
       <a class="activeClass"  href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Comment (scoopscode) [inline index.html:20]: Same from above, change to "UPDATES"

File: privacy.html [hunk 4]
```diff
@@ -18,6 +18,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Comment (scoopscode) [inline privacy.html:21]: Same as above, change to "UPDATES"

File: pull_requests.html [hunk 5]
```diff
@@ -17,6 +17,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="activeClass" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Comment (scoopscode) [inline pull_requests.html:20]: Same as above, change to "UPDATES"

File: tos.html [hunk 6]
```diff
@@ -16,6 +16,7 @@
       <a class="navbarlink" href="index.html">HOME</a>
       <a class="navbarlink" href="pull_requests.html">PULL REQUESTS</a>
       <a class="navbarlink" href="downloads.html">DOWNLOADS</a>
+      <a class="navbarlink" href="aircraft.html">AIRCRAFT UPDATES</a>
```
Comment (scoopscode) [inline tos.html:19]: Same as above, change to "UPDATES"

**Review by scoopscode — CHANGES_REQUESTED** [review]

File: aircraft.html (hunk 2 above)
Comment (Spherrrical) [reply aircraft.html:45]: We are allowed copyright under the GPLv3 License. You can check description for their copyright notice.

**Comment by Spherrrical** [comment]
Everything has been changed to current

**Review by scoopscode — APPROVED** [review]

**Pull request merged**